### DISCUSSION TITLE
ImageUploader 是否能够增加上传事件

### DISCUSSION THREAD

**Discussion by hugejile** [ant-design/ant-design-mobile#5121] [Q&A] [answered]
我需要拦截点击上传按钮，某些情况下调用浙里办的拍照事件，目前是不是没有办法实现？

**Comment by awmleer** [comment, marked as answer]
目前应该是比较难支持的，或许可以后面给 ImageUploader 加个 `onUploadClick` 属性，让用户直接拦截原生时间？

**Reply by awmleer** [reply]
我开了个 issue #5123 ，可以后面关注/订阅一下这个 issue
